packages/golden_toolkit/lib/src/golden_builder.dart

```diff
@@ -141,171 +121,39 @@ class GoldenBuilder {
       mainAxisSpacing: 16,
       shrinkWrap: true,
       crossAxisCount: columns,
-      children: tests,
+      children: scenarios,
     );
   }
 
-  Column _column() => Column(mainAxisSize: MainAxisSize.min, children: tests);
-}
-
-bool _inGoldenTest = false;
```

Comment:
moved the code related to testGoldens into testing_tools (no other changes)
moved the code related to multiScreenGolden into a new file